src/components/navbar/navbar.css

```diff
@@ -4,4 +4,35 @@ nav {
 
 .defineCenter {
     float: none !important;
+}
+
```

Comment:
Alert that searchBtn ("Go!" Button) when hover, the cursor is not turned into pointer, this should be added and amended.
![image](https://user-images.githubusercontent.com/59509722/105933488-74b90280-6089-11eb-9084-dfa50bce81ea.png)

Reply:
SearchBox have a line which should not be there
![image](https://user-images.githubusercontent.com/59509722/105933470-6cf95e00-6089-11eb-9deb-5c0d3b56adfe.png)